collections/_blogs/2020-07-22-meshery-accepted-into-cncf-landscape.md

```diff
@@ -21,16 +21,16 @@ Digging deeper into what service meshes had to offer I discovered Meshery - the
 
 Being a member of Layer5 was one of the nicest community experiences I've ever had. Mainly thanks to community's founder and captain - Lee Calcote. He always makes sure everybody feels significant, included and motivated to participate. And this spirit spreads out to all community activities.
 
-In no time I had my first commits approved and merged and started joining weekly community and development video calls.
+In no time, I had my first commits approved and merged and started joining weekly community and development video calls.
 
 #### We're on CNCF Landscape!
 
-Now - I won't be reviewing Meshery in this post. I've spoken about it in this video and some great introductory articles by community members can be found here and here. This post is more of a celebration - because last week **Meshery finally got featured in the CNCF landscape ‘Service Mesh’ category!**
+Now - I won't be reviewing Meshery in this post. I've spoken about it in this video and some great introductory articles by community members can be found here and here. This post is more of a celebration - because last week **Meshery now featured in the CNCF Landscape's ‘Service Mesh’ category!**
```

Comment:
>This post is more of a celebration - because last week **Meshery now featured in the CNCF Landscape's ‘Service Mesh’ category!**

This sounds wrong. Can I change it to, "This post is more of a celebration - because last week **Meshery got featured in the CNCF Landscape's ‘Service Mesh’ category!**" ?